.github/actions/lint-go-tfe/action.yml

```diff
@@ -30,15 +30,7 @@ runs:
 
     - name: Install mockgen
       shell: bash
-      run: |
-        set -eux -o pipefail
-        MOCKGEN_VERSION=$(curl -Ls -o /dev/null -w %{url_effective}   https://github.com/golang/mock/releases/latest | awk -F'/v' '{printf$2}')
-        MOCKGEN_ZIP=mock_${MOCKGEN_VERSION}_linux_amd64.tar.gz
-        curl -OL https://github.com/golang/mock/releases/download/v$MOCKGEN_VERSION/$MOCKGEN_ZIP
-        sudo tar xvzf $MOCKGEN_ZIP --strip-components 1 -C /usr/local
-        sudo chmod +x /usr/local/mockgen
-        rm -f $MOCKGEN_ZIP
-        echo /usr/local/ >> $GITHUB_PATH
+      run: go install github.com/golang/mock/mockgen@v1.6.0
```

Comment:
This was needed to help `mockgen` parse generics. It seems that installing it manually caused it to reference an older version of go?